### PR TITLE
VT-8 Entities

### PR DESCRIPTION
**Pull request by gbusila**

### CODE REVIEW

**Review by ccaraian — COMMENTED** [review]
Try to put to each field annotation @Column, ex: 

```
@Column(name = "name")
private String name
```

also add please JPA Repository for this entity, ex:
https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa

File: src/main/java/com/vesta/service/UserService.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserConverter userConverter;
+
+    public UserDto getById(Long id) {
+        return userConverter.converter(userRepository.findById(id).orElseThrow(null));
```
Comment (ccaraian) [inline src/main/java/com/vesta/service/UserService.java]: orElseThrow is used to throw an exception, in your case you need to use simple orElse

**Review by ccaraian — APPROVED** [review]

File: src/main/java/com/vesta/service/converter/UserConverter.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import com.vesta.repository.entity.UserEntity;
+import com.vesta.service.dto.UserDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserConverter {
+
+    public UserDto converter(UserEntity entity) {
+        UserDto dto = new UserDto();
```
Comment (ccaraian) [inline src/main/java/com/vesta/service/converter/UserConverter.java:12]: entity can be null so add if(entity == null) return null; 
the same for the second converter

**Review by vadimeladii — APPROVED** [review]
Good job

**Pull request merged**